mne/gui/tests/test_coreg_gui.py

```diff
@@ -146,22 +153,42 @@ def test_coreg_model():
     finally:
```

Comment:
This does not display the way I wanted, I mean this test: https://github.com/Eric89GXL/mne-python/blob/tweak-coreg/mne/gui/tests/test_coreg_gui.py#L152